lib/plugins/aws/package/compile/events/apiGateway/lib/validate.js

```diff
@@ -359,6 +359,9 @@ module.exports = {
     };
 
     if (typeof http.cors === 'object') {
+      if (typeof http.cors.methods === 'string') {
+        http.cors.methods = [http.cors.methods];
+      }
       cors = http.cors;
```

Comment:
http.cors.methods can be a string array ['GET', 'POST'] but also can be only one string like 'GET',
but the code, ex.: line 394 `cors.methods.push('OPTIONS');`
uses cors.methods as an array because of the .push() method,
so we need to ensure that cors.methods is an array